.swiftlint.yml

```diff
@@ -0,0 +1,88 @@
+excluded:
+  - Scripts
+  - Package.swift
+  - DocsSnippets
+  - Dependencies
+  - Sources/StreamChatUI/Generated
+  - Sources/StreamChatUI/StreamSwiftyGif
+  - Sources/StreamChatUI/StreamNuke
+  - Sources/StreamChat/StreamStarscream
+  - vendor
+  - UISDKdocumentation
+  - Tests
+  - Sources/StreamChatUI/Deprecations.swift
+  - Sources/StreamChatUI/**/**/AttachmentViewCatalog.swift
+  - Sources/StreamChat/Controllers/DataController.swift
+
+disabled_rules:
+  - large_tuple
+  - multiple_closures_with_trailing_closure
+  - todo
+  - function_parameter_count
+  - force_cast
+  - force_try
```

Comment:
Why are we disabling these ones? 🤔

Reply:
As mentioned above, this is also widespread - my thinking is to get this covered in a new ticket, just otherwise this PR will become huge WDYT?

Reply:
Makes sense 🙂